logs-shipper/src/main/java/jettyjersey/Parser.java

```diff
@@ -0,0 +1,19 @@
+package jettyjersey;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class Parser {
```

Comment:
What does it parse? `QueryStringParser` is better

Reply:
done